src/nebula/NebulaInstance.cpp

```diff
@@ -112,7 +112,7 @@ folly::Optional<int32_t> NebulaInstance::getPid(bool skipCache) {
 
 folly::Optional<int32_t> NebulaInstance::getIntFromConf(const std::string& key) const {
     try {
-        CHECK(conf_.isObject());
+        // CHECK(conf_.isObject());
```

Comment:
Why delete it?

Reply:
when the config initial failed it will return `folly::none` and print out some error message.